### PR TITLE
New postman file

### PR DESCRIPTION
**Pull request by ChristopherDN**
What kind of change does this PR introduce?

- [ ] `bug-fix`
- [ ] `new-feature`
- [x] `enhancement`

### CODE REVIEW

File: Docs/Payloads/Postman commands/Products/Kijani Product.postman_collection.json [hunk 1]
```diff
@@ -6,100 +6,103 @@
...
 		},
 		{
-			"name": "Get product by id",
+			"name": "Read all products",
 			"request": {
 				"method": "GET",
 				"header": [],
 				"url": {
-					"raw": "http://localhost:8080/api/products/{id}",
+					"raw": "http://localhost:8080/api/products",
```
Comment (MathiasReker) [inline Docs/Payloads/Postman commands/Products/Kijani Product.postman_collection.json]: ???

File: Docs/Payloads/Postman commands/Products/Kijani Product.postman_collection.json [hunk 2]
```diff
@@ -6,100 +6,103 @@
 	},
 	"item": [
 		{
-			"name": "Get all products",
```
Comment (Geofery) [inline Docs/Payloads/Postman commands/Products/Kijani Product.postman_collection.json:9]: Hvorfor Local Toast 8080???

**Comment by Geofery** [comment]
Hvorfor Local Toast 8080???

**Review by Geofery — COMMENTED** [review]
Good job

**Pull request closed without merging**